### PR TITLE
minor changes in the About page

### PR DESCRIPTION
**Pull request by Kushdapush**
made some ui changes in the about page

### CODE REVIEW

File: public/aboutPage/AboutPageImage1.jpg
Comment (mittalvansh) [inline public/aboutPage/AboutPageImage1.jpg:1]: Convert the image to WebP format and apply compression to enhance its file size optimization.

File: app/about/page.jsx
Comment (mittalvansh) [inline app/about/page.jsx:1]: Here, use next/Image for img

**Review by mittalvansh — CHANGES_REQUESTED** [review]

File: app/about/page.jsx
Comment (Kushdapush) [reply app/about/page.jsx:1]: Hey @mittalvansh PTAL

**Comment by mittalvansh** [comment]
@Kushdapush solve merge conflicts

**Review by mittalvansh — APPROVED** [review]

**Pull request merged**